make/tools.mk

```diff
@@ -527,7 +524,7 @@ endif
 ifeq ($(shell [ -d "$(ARM_SDK_DIR)" ] && echo "exists"), exists)
   ARM_SDK_PREFIX := $(ARM_SDK_DIR)/bin/arm-none-eabi-
 else
-  ifneq ($(MAKECMDGOALS),arm_sdk_install)
+  ifeq (,$(findstring _install,$(MAKECMDGOALS)))
```

Comment:
_sdk_ might be a better search term here, allowing cleanup etc. as well rather than just install.